jaeger-core/src/main/java/io/jaegertracing/JaegerSpanContext.java

```diff
@@ -16,12 +16,13 @@
 
 import io.jaegertracing.exceptions.EmptyTracerStateStringException;
 import io.jaegertracing.exceptions.MalformedTracerStateStringException;
+import io.opentracing.SpanContext;
```

Comment:
Q: was the refactoring done using IDE type renaming, or via search/replace?  If the latter, it's hard to see if there are any unintended API changes in this diff since the previous type of `SpanContext` is now changed to OT via import, without showing up in the diff.

Reply:
Initially, via the IDE, but I then changed some (most?) places to use OT's SpanContext. Based on our conversation, I later reverted this last step (also manually). The rule of thumb is that there should be no "semantic" change: where it was previously returning OT's Span Context should still return it and vice-versa.